wom/src/test/scala/wdl/wom/WdlSubworkflowWomSpec.scala

```diff
@@ -18,7 +18,7 @@ class WdlSubworkflowWomSpec extends FlatSpec with Matchers {
         |  Int x
         |  call import_me.inner as inner { input: i = x }
         |  output {
-        |    Array[String] out = inner.out
+        |    Array[String] out = [inner.out]
```

Comment:
It's awesome that these can now be statically type-checked!

TBH I think I'd prefer just fixing the type to `String` rather than adding the `[ ]`s, but if that means a lot of expectation fixing then I'm not too fussed

Reply:
The magic conversion of a single `Thing` to an `Array[Thing]` was a feature someone explicitly asked for way back when. I never liked this feature but I'm sure there's WDL out there that expects this to work.